docs/configuration.md

```diff
@@ -291,9 +291,9 @@ to match a complete time interval, all fields must match.
 Some fields support ranges and negative indices, and are detailed below. All definitions are
 taken to be in UTC, no other timezones are currently supported.
 
-`time_range` Ranges inclusive of the starting time and exclusive of the end time to
+`time_range` ranges are inclusive of the starting time and exclusive of the end time to
```

Comment:
I think maybe this just needs a colon after the ‘time_range’ part? The other descriptions uses the section name as a heading, rather than using them as part of the description.

Reply:
Ahhh, that's how that was meant, yeah. Pushed a change to just add the semicolon here.